R/pkg/R/client.R

```diff
@@ -42,6 +42,19 @@ determineSparkSubmitBin <- function() {
   }
   sparkSubmitBinName
 }
+# R supports both file separator in the file path (Unix : / and Windows: \) irrespective of the operating system
+# but when passing file path to Java or script program, it has to be converted according to operating system
+
+determinefileSeperator <- function() {
+  if (.Platform$OS.type == "unix") {
```

Comment:
@sun-rui @JoshRosen  I have a window machine. Although I am not used to R, if changing the condition to `.Platform$OS.type != "windows"` and rebasing are only things to be done, I can make a PR based on this. (The author might have to be @prakashpc though).